environments/staging/aws-resources.yml

```diff
@@ -40,10 +42,6 @@ pgbouncer0-staging.instance_id: i-0315ba6f2f8ba45f5
 pgformplayer0-staging: pgformplayer0-staging.czkracjslrn2.us-east-1.rds.amazonaws.com
 pgformplayer_nlb-staging: pgformplayer-nlb-staging-98b68467f6f5717b.elb.us-east-1.amazonaws.com
 pgmain_nlb-staging: pgmain-nlb-staging-fe77ab2d143eedbb.elb.us-east-1.amazonaws.com
-pgproxy2-staging: 10.201.40.16
-pgproxy2-staging.instance_id: i-0638ff6218ed9f753
-pgproxy3-staging: 10.201.40.79
-pgproxy3-staging.instance_id: i-0ceb5822e286daf7e
```

Comment:
assuming this is just unrelated cleanup?

Reply:
Yes, it came with `aws-fill-inventory`, probably a leftover from another PR/maintenance.